Project2/style.css

```
@@ -0,0 +1,25 @@
+.card {
+    max-width: 500px;
+    margin: 10px auto;
+    background-color: #ffffff;}
+figure {margin: 0 auto;}
+        max-width: 500px;
+.button1 {
+    background-color: #008CBA;
+    border: none;
+    color: white;
+    padding: 15px 32px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+  }
+  body {margin: 10px
+    }
+    .thumbnail {
+        max-width: 500px; 
+    }\
+p{margin-left: 10; margin-left: 10px}
+.words {  padding: 10px
```

This declaration is missing its closing semicolon:
```suggestion
.words {
  padding: 10px;
```